internal/handlers/pg/msg_aggregate.go

```diff
@@ -91,10 +93,64 @@ func (h *Handler) MsgAggregate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 		v = int64(0)
 	}
 
-	maxTimeMS, err := commonparams.GetValidatedNumberParamWithMinValue(document.Command(), "maxTimeMS", v, 0)
+	// cannot use other existing commonparams function, they return different error codes
+	maxTimeMS, err := commonparams.GetWholeNumberParam(v)
 	if err != nil {
-		// unreachable for MongoDB GO driver, it validates maxTimeMS parameter
-		return nil, lazyerrors.Error(err)
+		switch {
```

Comment:
Is this switch-case the same as in `internal/handlers/common/getmore.go`? If so, maybe we can move the common code to some function and call it? (Though, it feels like the opposite of what we did a while ago...)

Reply:
Unfortunately they are not the same unfortunately, their only difference of error code is upon `commonparams.ErrLongExceededNegative` returns `ErrBadValue` for `getMore` while `ErrLongExceededNegative` for `aggregate`.

Reply:
Ah! Well, yes, then we have to keep both! Thanks for the explanation.